.travis.yml

```diff
@@ -1,39 +1,29 @@
 matrix:
-    include:
-        - language: cpp
-          sudo: false
-          compiler: clang
-          script:
-              - mkdir build && cd build
-              - cmake ../
-              - cmake --build .
-              - ctest --verbose .
-        - sudo: false
-          branches:
-              only:
-                  - master
-          addons:
-              apt:
-                  packages:
-                      - doxygen
-          script:
-              - cd docs && doxygen && cd ..
-          deploy:
-              provider: pages
-              skip_cleanup: true
-              local_dir: docs/html
-              github_token: $GH_REPO_TOKEN
-              on:
-                  branch: master
-        - language: python
-          python:
-              - "3.6"
-              - "3.7"
-              - "3.8"
-          sudo: false
-          install:
-              - pip install -r requirements.txt
-              - pip install -vvv .
-          script:
-              - pytest -v test_lin.py
-
+  include:
+    - language: cpp
+      addons:
+      apt:
+        sources:
+          - ubuntu-toolchain-r-test
+        packages:
+          - bazel
+      sudo: false
+      compiler: clang
+      before_install:
+        - wget https://github.com/bazelbuild/bazel/releases/download/3.4.1/bazel_3.4.1-linux-x86_64.deb.sha256
+        - wget https://github.com/bazelbuild/bazel/releases/download/3.4.1/bazel_3.4.1-linux-x86_64.deb
+        - sha256sum -c bazel_3.4.1-linux-x86_64.deb.sha256
+        - sudo dpkg -i bazel_3.4.1-linux-x86_64.deb
+      script:
+        - bazel test //test:all --verbose_failures
+    - language: python
+      python:
+        - "3.6"
+        - "3.7"
+        - "3.8"
+      sudo: false
+      install:
+        - pip install -r requirements.txt
+        - pip install -vvv .
+      script:
+        - pytest -v test_lin.py
```

Comment:
You can also work this test item into Bazel if you want.

Reply:
Bazel is fucking magical

Reply:
Will look into it in the future!